app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TasksController < ApiController
+      def create
+        created_task = Task.create!(create_params)
+        render json: { id: created_task.id }
+      end
+
+      private
+
+      def create_params
+        params.permit(:name, :description, :due_at)
```

Comment:
Tá rolando esse erro aqui:
```
Unpermitted parameter: :task. Context: { controller: Api::V1::TasksController, action: create, request: #<ActionDispatch::Request:0x00007f7d2aeb7db0>, params: {"name"=>"Do something new", "description"=>"Description", "controller"=>"api/v1/tasks", "action"=>"create", "task"=>{"name"=>"Do something new", "description"=>"Description"}} }
```

```suggestion
        params.require(:task).permit(:name, :description, :due_at)
```